saleor/wombat/serializers.py

```diff
@@ -0,0 +1,200 @@
+    tax = serializers.DecimalField(max_digits=9, decimal_places=2)
+    shipping = serializers.DecimalField(max_digits=9, decimal_places=2)
+    payment = serializers.DecimalField(max_digits=9, decimal_places=2)
+    order = serializers.DecimalField(max_digits=9, decimal_places=2)
+
+
+class LineItemsSerializer(serializers.ModelSerializer):
+    product_id = serializers.CharField()
+    name = serializers.CharField(source='product_name')
+    price = serializers.DecimalField(source='unit_price_gross', max_digits=9,
+                                     decimal_places=2)
+
+    class Meta:
+        model = OrderedItem
+        fields = ('product_id', 'name', 'price', 'quantity')
+
+
+class OrderSerializer(serializers.ModelSerializer):
+    billing_address = AddressSerializer()
+    shipping_address = AddressSerializer()
+    email = serializers.EmailField(source='get_user_email')
+    placed_on = serializers.DateTimeField(source='created')
+    totals = serializers.SerializerMethodField()
+    channel = serializers.SerializerMethodField()
+    adjustments = serializers.SerializerMethodField()
+    payments = PaymentsSerializer(many=True)
+    line_items = LineItemsSerializer(many=True, source='items')
+
+    def get_totals(self, order):
+        return {
+            'item': sum(item.unit_price_gross*item.quantity
+                        for item in order.items.all()),
+            'adjustment': order.get_total().tax,
+            'tax': order.get_total().tax,
+            'shipping': order.get_delivery_total().gross,
+            'payment': sum(p.total for p in order.payments.all()),
+            'order': order.get_total().gross
+        }
+
+    def get_adjustments(self, order):
+        return [{
+            'name': 'Tax',
+            'value': order.get_total().tax
+        }]
+
+    def get_channel(self, order):
+        return 'Saleor'
+
+    class Meta:
+        model = Order
+        fields = (
+            'id',
+            'status',
+            'channel',
+            'email',
+            'currency',
+            'placed_on',
+            'totals',
+            'adjustments',
+            'line_items',
+            'shipping_address',
+            'billing_address',
+            'payments'
+        )
+
+
+class ProductVariantSerializer(serializers.ModelSerializer):
+
+    price = serializers.SerializerMethodField()
+    cost_price = serializers.SerializerMethodField()
+    quantity = serializers.SerializerMethodField()
+    options = serializers.SerializerMethodField()
+
+    def get_price(self, obj):
+        return obj.get_price().gross
+
+    def get_cost_price(self, obj):
+        # Get maximal cost price for a product
+        stock = sorted(obj.stock.all(), key=lambda s: s.cost_price,
+                       reverse=True)
+        return stock[0].cost_price
+
+    def get_quantity(self, obj):
+        # Get maximal quantity for a product
+        stock = sorted(obj.stock.all(), key=lambda s: s.quantity,
```

Comment:
Isn't this available somewhere else already? This is business logic, we don't want to have multiple implementations.